docs/manifest.json

```diff
@@ -143,6 +143,12 @@
 		"markdown_source": "https:\/\/raw.githubusercontent.com\/WordPress\/gutenberg\/master\/docs\/release.md",
 		"parent": "reference"
 	},
+	{
+		"title": "Repository Management",
+		"slug": "release",
```

Comment:
Won't this slug conflict with the `Release Process` (release.md) one above?

Reply:
Good catch. Updated in 87c867880b363bb5f08b8a24f76c3a8d63e01a34